pkg/generators/markers.go

```diff
@@ -87,6 +113,10 @@ func (c CommentTags) Validate() error {
 		err = errors.Join(err, fmt.Errorf("multipleOf cannot be 0"))
 	}
 
+	for _, celTag := range c.CEL {
+		err = errors.Join(err, celTag.Validate())
```

Comment:
I think it would be also useful to indicate the index of the expression that failed. If a user accidentally skips an index, it's quite difficult to pinpoint.